src/view/aktivitet/SamtalereferatAktivitet.tsx

```diff
@@ -1,23 +1,34 @@
 import React from 'react';
 
+import { SamtalereferatIkkeDelt } from '../../felleskomponenter/etiketer/Etikett';
 import { Aktivitet } from '../../utils/aktivitetTypes';
 import { formaterDate } from '../../utils/Date';
-import InformasjonElement from './InformasjonElement';
+import { useUserInfoContext } from '../Provider';
+import InformasjonElement, { InformasjonElementRaw } from './InformasjonElement';
 import { getKanalTekst } from './MoteAktivitet';
 
 interface PropTypes {
     aktivitet: Aktivitet;
 }
 
-//TODO: Ok med referat her? har en annen type visning i aktivitetsplanen
 export default function SamtalereferatAktivitet(props: PropTypes) {
-    const { fraDato, kanal, referat } = props.aktivitet;
+    const { fraDato, kanal, referat, erReferatPublisert } = props.aktivitet;
+
+    const bruker = useUserInfoContext();
+    const erVeileder = !!bruker?.erVeileder;
+    const referatPublisertElement = <InformasjonElement merkelapptekst="Referat" verdi={referat} />;
+    const referatIkkePublisertElement = erVeileder ? (
+        <InformasjonElementRaw merkelapptekst="Referat">
+            {SamtalereferatIkkeDelt({ visible: true })}
+        </InformasjonElementRaw>
+    ) : null;
```

Comment:
Tror dette også kan skrives sånn her: 
```suggestion
    const referatIkkePublisertElement = erVeileder && (
        <InformasjonElementRaw merkelapptekst="Referat">
            {SamtalereferatIkkeDelt({ visible: true })}
        </InformasjonElementRaw>
    );
```